indico/modules/events/papers/blueprint.py

```diff
@@ -39,6 +39,8 @@
 _bp.add_url_rule('/manage/papers/assignment-list/', 'assignment', management.RHPapersAssignmentList)
 _bp.add_url_rule('/manage/papers/assignment-list/customize', 'customize_assignment_list',
                  management.RHAssignmentListCustomize, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/papers/assignment-list/change-state', 'manage_papers_state',
```

Comment:
Don't we call this "judging" in the abstracts module (RH name, URL, endpoint name, etc)? In that case I'd use similar naming here for consistency.